wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/SchemaHandler.kt

```diff
@@ -65,13 +63,8 @@ abstract class SchemaHandler {
    * A [Context] holds the information necessary for a [SchemaHandler] to do its job. It contains
    * both helping objects such as [logger], and constraining objects such as [emittingRules].
    */
-  data class Context(
-    /** To be used by the [SchemaHandler] for reading/writing operations on disk. */
-    val fileSystem: FileSystem,
-    /** Location on [fileSystem] where the [SchemaHandler] is to write files, if it needs to. */
-    val outDirectory: Path,
-    /** Event-listener like logger with which [SchemaHandler] can notify handled artifacts. */
-    val logger: WireLogger,
```

Comment:
As mentioned above, I think we want to keep this and remove the `logArtifactHandled` method from `FireWriter`.